server/models/message.coffee

```diff
@@ -571,6 +592,8 @@ Message.createFromImapMessage = (mail, box, uid, callback) ->
                 name: att.generatedFileName
                 buffer: buffer
 
+        delete mail.attachments
```

Comment:
var a = {}
Object.keys(a) ---> Array [  ]
a.test = 'value'
Object.keys(a) ---> Array [ "test" ]
a.test = undefined
Object.keys(a) ---> Array [ "test" ]
delete a.test
Object.keys(a)   --->  Array [  ]
It is faster for "hot" objects (it prevents some interpreter optimization with hidden class), but its not exactly the same. I'd rather not risk introducing subtle bugs for a µs improvement of a once-called route.